src/uct/ugni/udt/ugni_udt_iface.c

```diff
@@ -479,8 +466,6 @@ static UCS_CLASS_INIT_FUNC(uct_ugni_udt_iface_t, uct_md_h md, uct_worker_h worke
         ucs_warn("GNI_EpDestroy failed, Error status: %s %d",
                  gni_err_str[ugni_rc], ugni_rc);
     }
- clean_iface_activate:
-    ugni_deactivate_iface(&self->super);
```

Comment:
In a different commit that apparently I failed to include. Oops, will rectify that.